doc/examples/date-string-example.js

```diff
@@ -1,4 +1,4 @@
-import {DefineMap, QueryLogic, Reflect as canReflect} from "can";
+import {DefineMap, QueryLogic } from "can";
```

Comment:
why a space at the end and not one at the beginning?

Reply:
Find + replace mistake, fixed now.